### DISCUSSION TITLE
Failed API call

### DISCUSSION THREAD

**Discussion by Ziko456** [ucfcdl/UDOIT#743] [Q&A]
I currently have UDOIT installed on Heroku. For some reason I keep getting an error on the welcome page whenever I access it using the UDOIT dashboard on Canvas. "Failed API call" keeps popping up and nothing is accessible on that welcome page. I've also tried installing UDOIT in our on premise environment and got the same result. I've created the API key and LTI key per UDOIT documentation. Please let me know if anyone has experienced/seen that error and know if there is a fix for it. Thanks in advance!
![API_fail](https://user-images.githubusercontent.com/41217665/150160198-2edd579e-84a7-4e1d-8352-94cd6d3f3b0d.PNG)
![UDOIT_error](https://user-images.githubusercontent.com/41217665/150160223-fd408039-f0b1-4092-9164-680baa822788.PNG)

**Comment by rob-3** [comment]
Sorry; just noticed this. Can you reproduce this with the `dev/v3-2-0` branch?
